internal/command/state_identities_test.go

```diff
@@ -448,10 +448,6 @@ func TestStateIdentities_stateStore(t *testing.T) {
 		"default": stateBytes,
 	}
 	mockProviderAddress := addrs.NewDefaultProvider("test")
-	providerSource, close := newMockProviderSource(t, map[string][]string{
-		"hashicorp/test": {"1.0.0"},
-	})
-	defer close()
 
```

Comment:
Context: I realised that in lots of tests for PSS I've been adding this provider source value into the Meta in tests. Provider sources are only needed in the context of an init command, so providing them in these tests is unnecessary.

Similarly there are times where supplying both a `View` and `Ui` is unnecessary, and some commands need a `Stream`.  I guess this reflects how different commands' terminal output is implemented slightly differently (shout out to https://github.com/hashicorp/terraform/issues/37439), and so only some outputs are required in their tests.